### PR TITLE
LCH 6790 3.4.x : Port maintenance mode changes

### PR DESCRIPTION
**Pull request by kirtigarg2584**
**Motivation**

Fixes #NNN

**Proposed changes**

**Alternatives considered**

**Testing steps**

### CODE REVIEW

**Review by narendradesai — APPROVED** [review]
Added test coverage. Looks fine

File: src/StatusCodes.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+/**
+ * Contains status codes returned by the service.
+ */
+final class StatusCodes {
+
+  /**
+   * Returned when the service is intentionally configured to be in maintenance.
+   */
+  public const SERVICE_UNDER_MAINTENANCE = 503;
```
Comment (saxumVermes) [inline src/StatusCodes.php:13]: That's the wrong maintenance code @narendradesai @kirtigarg2584

Comment (saxumVermes) [reply src/StatusCodes.php:13]: It is supposed to be `5031`

**Comment by saxumVermes** [comment]
This PR needs to be reopened @kirtigarg2584

**Comment by narendradesai** [comment]
@saxumVermes @kirtigarg2584  Created PR to update maintenance mode code: https://github.com/acquia/content-hub-php/pull/190

**Pull request merged**